k8s/helmfile/env/production/elasticsearch-2.values.yaml.gotmpl

```diff
@@ -70,3 +70,10 @@ ingest:
   enabled: false
 
 clusterName: {{ .Release.Name }}
+
+volumePermissions:
```

Comment:
Assuming we're setting this so if we use this image in the future we won't be bitten by the same thing. From what I could see we don't currently use this one.